src/Bindings/ManualBindings_World.cpp

```diff
@@ -380,9 +378,9 @@ static int tolua_cWorld_QueueTask(lua_State * tolua_S)
 		return cManualBindings::lua_do_error(tolua_S, "Error in function call '#funcname#': Could not get function reference of parameter #1");
 	}
 
-	auto task = std::make_shared<cLuaWorldTask>(*Plugin, FnRef);
-	Plugin->AddResettable(task);
-	self->QueueTask(task);
+	auto ResettableTask = std::make_shared<cLuaWorldTask>(*Plugin, FnRef);
+	Plugin->AddResettable(ResettableTask);
+	self->QueueTask(std::bind(&cLuaWorldTask::Run, ResettableTask, std::placeholders::_1));
```

Comment:
I can't quite get std::fuction to work. Do you have any ideas that are one-liners?

It still seems to me the std::bind is the best way. The complaints thus far has been entirely of aesthetics, convention, and a fear of bind - do correct me if I'm wrong here. If it works, and it does excellently, why not use it?